mpfilepicker/src/macosX64Main/kotlin/com/darkrockstudios/libraries/mpfilepicker/FilePicker.macos.kt

```diff
@@ -118,3 +121,30 @@ public actual fun DirectoryPicker(
 		}
 	}
 }
+
+@Composable
+public actual fun SaveFilePicker(
+	show: Boolean,
+	title: String?,
+	path: String?,
+	filename: String,
+	fileExtension: String?,
+	onFileSelected: FileSelected,
+) {
+	// title, prompt, message, nameFieldLabel
+	LaunchedEffect(show) {
+		if (show) {
+			with(NSSavePanel()) {
+				if (path != null) directoryURL = NSURL.fileURLWithPath(path, true)
+				canCreateDirectories = true
+				nameFieldStringValue = filename
+				if (fileExtension != null) message = fileExtension
```

Comment:
Message is the title of the dialog. Don't set fileExtension to it.

```suggestion
				if (fileExtension != null) message = title
```